Makefile

```
@@ -0,0 +1,57 @@
+# Tauri Development Makefile
+# Common development tasks for the project
+
+.PHONY: help dev build clean lint format check test tauri-dev tauri-build
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)